Library/Source/RuntimeImpl.cpp

```diff
@@ -226,7 +228,7 @@ namespace Babylon
         m_env = &env;
         auto hostScopeGuard = gsl::finally([this] { m_env = nullptr; });
 
-        InitializeJavaScriptVariables();
+        InitializeJavaScriptVariables(600, 400);
```

Comment:
What's the motivation for committing this? IIRC, we originally started using the 32x32 so that it would be painfully obvious if it wasn't getting updated properly, since it's supposed to be reset to something else immediately.

Reply:
I'll change it back to 32x32